docker-compose-dev.yml

```diff
@@ -109,6 +109,17 @@ services:
     volumes:
       - './services/client:/usr/src/app'
 
+  lambdas-test:
+    container_name: lambdas-test
+    build:
+      context: ./lambdas
+      dockerfile: Dockerfile-test
+    environment:
+      - NODE_ENV=development
+      - CI=${CI:-false}
```

Comment:
Is the CI variable required to run linting on CI?

Reply:
@apoclyps Probably not (copy/paste from the client test run), but I'll leave it in place if that's ok to cover when this image is also used for the test run